source/features/tab-size.css

```diff
@@ -1,41 +1,16 @@
 /* Make tab-indented code more readable on GitHub and Gist */
 
-* {
-	tab-size: var(--tab-size) !important;
-}
-
-[data-tab-size='2'] {
-	--tab-size: 2;
-}
-
-:root, /* Changes the browser default */
-.comment-body [data-tab-size='8'], /* User setting is ignored in comments #4833 */
-[data-tab-size='4'] {
+:root, /* Changes the browser default, it applies to textareas too */
+.comment-body .tab-size[data-tab-size='8'] /* User setting is ignored in file blocks in comments #4833 */ {
 	--tab-size: 4;
+	tab-size: var(--tab-size);
 }
```

Comment:
Instead of this (setting `--tab-size` again inside comments), it may be better to have

```css
[data-tab-size='8']:not(.highlight) {
	--tab-size: 8;
}
```

Thoughts?

Reply:
That looks good as long as the final result is 4 characters in that situation, which is the point of `tab-size`

Reply:
Right, this just makes it easier to override `--tab-size`:

```diff
- .comment-body .tab-size[data-tab-size='8'],
  :root {
  	tab-size: var(--tab-size);
  }
```